pop-api/src/v0/assets/fungibles.rs

```diff
@@ -58,6 +58,83 @@ mod constants {
 	pub(super) const BURN: u8 = 20;
 }
 
+pub mod events {
```

Comment:
Reviewed! The PR looks good to me. But I think we should have a documentation for this somewhere in the code (or in a guidebook) to mention that using these events in the contract would make the contract size larger. And if not needed, the developers should make use of the runtime event that we already exposed by default.

Reply:
Link for reference: https://use.ink/basics/events#cost-of-using-events

Reply:
docs: events contract